packaging/windows/amazon-cloudwatch-agent-ctl.ps1

```diff
@@ -356,27 +356,27 @@ Function CWAConfig() {
     }
 
     if ($ConfigLocation -eq $AllConfig -And $multi_config -ne 'remove') {
-        Write-Output "ignore cwa configuration `"$AllConfig`" as it is only supported by action `"remove-config`""
+        Write-Output "Ignore amazon-cloudwatch-agent's configuration ${AllConfig} as it is only supported by action `"remove-config`""
         return
     }
 
     if ($ConfigLocation -eq $AllConfig) {
         Remove-Item -Path "${JSON_DIR}\*" -Force -ErrorAction SilentlyContinue
     } else {
-        & $CWAProgramFiles\config-downloader.exe --output-dir "${JSON_DIR}" --download-source "${ConfigLocation}" --mode "${param_mode}" --config "${COMMON_CONIG}" --multi-config "${multi_config}"
+        & cmd /c "`"$CWAProgramFiles\config-downloader.exe`" --output-dir ${JSON_DIR} --download-source ${ConfigLocation} --mode ${param_mode} --config ${COMMON_CONIG} --multi-config ${multi_config} 2>&1"
         CheckCMDResult
     }
 
     $jsonDirContent = Get-ChildItem "${JSON_DIR}" | Measure-Object
 
     if ($jsonDirContent.count -eq 0) {
-        Write-Output "all amazon-cloudwatch-agent configurations have been removed"
+        Write-Output "All amazon-cloudwatch-agent configurations have been removed"
         Remove-Item "${TOML}" -Force -ErrorAction SilentlyContinue
     } else {
         Write-Output "Start configuration validation..."
         & cmd /c "`"$CWAProgramFiles\config-translator.exe`" --input ${JSON} --input-dir ${JSON_DIR} --output ${TOML} --mode ${param_mode} --config ${COMMON_CONIG} --multi-config ${multi_config} 2>&1"
         CheckCMDResult
-        # Let command pass so we can check return code and give user-friendly error-message

```

Comment:
Why remove this comment? It is explaining what setting the `$ErrorActionPreference` does.

Reply:
Forgot to update the comment on the config-translator since the existing one would not be enough. However, I have fixed it in the newest commit.

Reply:
Its for better information right? Just wank to ask if there anything wrong with the new comments. Customer won't look at this file; therefore, its only for dev.

Reply:
I think if you want to include information, what needs to be explained is why the `ErrorActionPreference` is set here, and not in the line above it for the config translator. If there isn't a clear explanation for that, I don't see value in trying to add more comments here. I also don't see any reason to link to https://www.tutorialspoint.com/how-to-use-the-erroractionpreference-variable-in-powershell

Reply:
Make sense. I have updated the newest comment regarding these details. Thanks for reminding me and sorry for commenting late.